index.js

```diff
@@ -0,0 +1,18 @@
+const Discord = require('discord.js');
+require('dotenv').config();
```

Comment:
whats this for

Reply:
basically a .env file is a file used to store secrets (such as passwords or ids) that you don't want to publish publically. 
the contents of the file are literally a bunch of key val pairs like this:
<code>
SECRET_KEY = SECRET_VALUE
</code>

['dotenv'](url) is a small library that parses the .env file and makes the things in the file accessible as variables.

it actually does this by making them accessible via the global object <code>process.env</code>

e.g. <code>process.env.SECRET_KEK</code>

@bailslearnsstuff

Reply:
NICE